aptos-move/aptos-vm/src/zkid_validation.rs

```diff
@@ -38,79 +46,70 @@ fn get_jwks_onchain(resolver: &impl AptosMoveResolver) -> anyhow::Result<Patched
     Ok(jwks)
 }
 
+fn get_jwk_for_zk_authenticator(
```

Comment:
Note to self: How come we didn't have this function before? Maybe it was specialized for the OpenIdSig path.